docs/Users_Guide/refs.rst

```diff
@@ -130,6 +130,12 @@ References
 |   a review and proposed framework. *Meteorological Applications*, 15, 51-64.
 |
 
+.. _Ebert-Uphoff-2024:
+
+| Ebert-Uphoff, I.,, 2024: An Investigation of Metrics to Evaluate the Sharpness
```

Comment:
I wonder if this fits under "unpublished material" type. If so, that would mean it can be mentioned in-line with I. Ebert-Uphoff (2024, unpublished article) and is not included in the reference page.

However, if this is expected to be accepted soon, then I'd rather we just stick to a normal journal article citation (similar to how it currently is written) rather than trying to remember the name of the article later on.